skimage/color/tests/test_colorconv.py

```diff
@@ -491,9 +491,12 @@ def test_rgb2yiq_conversion(self):
 
 def test_gray2rgb():
     x = np.array([0, 0.5, 1])
+    w = gray2rgb(x)
 
-    with pytest.raises(ValueError):
-        gray2rgb(x)
+    assert_equal(w.shape, (3, 1, 3))
+    assert_equal(w.dtype, x.dtype)
+    assert_equal(w.shape[0], x.shape[0])
+    assert_equal(w[0, 0, :], [0, 0, 0])
```

Comment:
I had put in so many assertions just to be consistent with the other tests, but something simpler sounds reasonable.